CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [3.0.1] - 2023-09-13
+
+### Added
+
+- Funding files and updates in the Readme about funding the project.
```

Comment:
This is not functionality added to this package which affects end-users, so IMO doesn't belong in "Added".

Maybe mention it in "Changed" ?